.circleci/config.yml

```diff
@@ -141,7 +141,8 @@ workflows:
               only: release
       - release_dockerhub:
           requires:
-            - release_pip
+            - test_linux
+            - test_macos
```

Comment:
I think you might be a bit behind the upstream master branch. These release jobs no longer require the test jobs as of https://github.com/artsy/hokusai/pull/107.